modules/elements-color-picker/assets/js/editor/module.js

```diff
@@ -0,0 +1,9 @@
+import Component from './component';
+
+export default class ElementsColorPicker extends elementorModules.ViewModule {
+	onInit() {
+		super.onInit();
+
+		$e.components.register( new Component() );
```

Comment:
Better to use event `elementor:init-components` to register the components. if there is no better propose.
```suggestion
		$e.components.register( new Component() );
```